src/common/string/filter/sequence-matching.ts

```diff
@@ -59,6 +59,7 @@ export const fuzzyFilterSort: FuzzyFilterSort = (filter, items) => {
         : fuzzySequentialMatch(filter, item.text);
       return item;
     })
+    .filter((item) => item.score === undefined || item.score > 0)
```

Comment:
Why is this in here? This would return all invalid items.

Reply:
Fixed here: https://github.com/home-assistant/frontend/pull/7430

Reply:
That's your code 😅

Reply:
lol, it is? 😆 Man I need to learn to read git history better. Sigh...